### DISCUSSION TITLE
Adding associated anatomy

### DISCUSSION THREAD

**Discussion by bendichter** [dandi/helpdesk#7] [Q&A] [answered]
What is the recommend workflow for adding an associated anatomical region to a dandiset?

For instance, for [dandiset 3](https://dandiarchive.org/dandiset/000003/draft), I would like to associated the primary visual cortex of a mouse. I can get to this view:

![image](https://user-images.githubusercontent.com/844306/117339442-bb2eb200-ae6d-11eb-8927-8932697d12e1.png)

but it is unclear where to go from here. Are there certain search engines that can help me identify an ontological term? Once I get that, what is the correct way to enter this information here?

**Comment by satra** [comment, marked as answer]
http://www.ontobee.org/ontology/UBERON is probably a good place to start.

**Reply by bendichter** [reply]
Thanks, that's helpful. So fill this out like this?

![image](https://user-images.githubusercontent.com/844306/117342395-35ad0100-ae71-11eb-9992-f4215fa3aecd.png)

**Reply by satra** [reply]
yes, that works.
